src/Modules/ProjectModule/components/ProjectList/ProjectList.tsx

```diff
@@ -182,69 +196,83 @@ export default function ProjectList() {
-                          </li>
-                          <li
-                            role="button"
-                            onClick={() => {
-                              navigate("/dashboard/projects-data", {
-                                state: pro,
-                              });
-                            }}
-                            className="px-3 py-1"
-                          >
-                            <div role="button" className="dropdown-div">
-                              <i className="fa-regular fa-pen-to-square me-2 "></i>
-                              {window.innerWidth < 650 ? "" : <span>Edit</span>}
-                            </div>
-                          </li>
-                          <li
-                            role="button"
-                            onClick={() => handleShowDelete(pro.id, pro.title)}
-                            className="px-3 py-1 "
-                          >
-                            <div className="dropdown-div">
-                              <i className="fa-solid fa-trash-can me-2"></i>
-                              {window.innerWidth < 650 ? (
-                                ""
-                              ) : (
-                                <span>Delelte</span>
-                              )}
-                            </div>
-                          </li>
-                        </ul>
+                            <li
+                              role="button"
+                              // onClick={() => handleShowShowItem(pro)}
+                              className="px-3 py-1 pt-2  "
+                            >
+                              <div className="dropdown-div ">
+                                <i className="fa-regular fa-eye me-2"></i>
+                                {window.innerWidth < 650 ? (
+                                  ""
+                                ) : (
+                                  <span>View</span>
+                                )}
+                              </div>
+                            </li>
+                            <li
+                              role="button"
+                              onClick={() => {
+                                navigate("/dashboard/projects-data", {
+                                  state: pro,
+                                });
+                              }}
+                              className="px-3 py-1"
+                            >
+                              <div role="button" className="dropdown-div">
+                                <i className="fa-regular fa-pen-to-square me-2 "></i>
+                                {window.innerWidth < 650 ? (
+                                  ""
+                                ) : (
+                                  <span>Edit</span>
+                                )}
+                              </div>
+                            </li>
+                            <li
```

Comment:
![image](https://github.com/OmarBazeed/project-mangement-system/assets/64327685/c7213c41-87f6-46aa-9c3b-796a9febfa43)

Reply:
We meed to make it a button